src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,46 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  userId: number;
+  selectedTodoId: Todo[];
+  selectedTodo: (value: number) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  userId,
+  selectedTodoId,
+  selectedTodo,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [todoCard, setTodoCard] = useState(false);
```

Comment:
what is todoCard? Why did you create it?

Reply:
i re-named it to better understanding